src/test/java/core/basesyntax/separator/FruitTransactionsParserImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.separator;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.FruitShopStorage;
+import core.basesyntax.model.Transaction;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.Test;
+
+public class FruitTransactionsParserImplTest {
+    private static final List<String> validTestData = List.of(
+            "type,fruit,quantity",
+            "b,apple,100",
+            "b,banana,100");
+    private static final List<String> invalidTestData = List.of(
+            "type,fruit,quantity",
+            "b,100,apple");
+    private static final int FIRST_INDEX = 0;
+    private static final int SECOND_INDEX = 1;
+    private static final FruitTransactionsParser parser = new FruitTransactionsParserImpl();
```

Comment:
```suggestion
    private static final List<String> validTestData;
    private static final List<String> invalidTestData;
    private static final FruitTransactionsParser parser = new FruitTransactionsParserImpl();
```
initialize in method with annotation @BeforeClass